plugins/core/tracer_ignore.go

```diff
@@ -0,0 +1,144 @@
+// Licensed to Apache Software Foundation (ASF) under one or more contributor
+// license agreements. See the NOTICE file distributed with
+// this work for additional information regarding copyright
+// ownership. Apache Software Foundation (ASF) licenses this file to you under
+// the Apache License, Version 2.0 (the "License"); you may
+// not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package core
+
+import (
+	"strings"
+)
+
+const (
+	PERIOD = "."
+	COLON  = ":"
+)
+
+func tracerIgnore(operationName string, ignoreSuffixList []string, ignorePath []string) bool {
+	return ignoreSuffix(operationName, ignoreSuffixList) || traceIgnorePath(operationName, ignorePath)
+}
+
+func ignoreSuffix(operationName string, ignoreSuffix []string) bool {
+	suffixIdx := strings.LastIndex(operationName, PERIOD)
+	if suffixIdx == -1 {
+		return false
+	}
+	for _, suffix := range ignoreSuffix {
+		if suffix == operationName[suffixIdx:] {
+			return true
+		}
+	}
+	return false
+}
+
+func traceIgnorePath(operationName string, ignorePath []string) bool {
+	op := strings.Index(operationName, COLON)
```

Comment:
Why need to check the operation name must-have :, if the operation is generated by another RPC framework(not HTTP), then this configuration will be broken.

Reply:
Are you referring to the occurrence of an empty value for operationName? I'm currently conducting validation based on the operationName dimension.

Reply:
Nope, I mean why the operation name must-have `:`? Not all operation names contain this symbol. It could make users confused as to why It does not work for other span names.

Reply:
I did overlook this issue and was only considering web requests before, I will adjust my strategy.